ampl-converter/src/test/java/com/powsybl/ampl/converter/AmplNetworkWriterTest.java

```diff
@@ -272,17 +275,29 @@ void writeHeadersWithUnknownVersion() {
         properties.put("iidm.export.ampl.export-version", "V1_0");
 
         Exception e = assertThrows(IllegalArgumentException.class, () -> export(network, properties, dataSource));
-        assertTrue(e.getMessage().contains("Value V1_0 of parameter iidm.export.ampl.export-version is not contained in possible values [1.0]"));
+        assertTrue(e.getMessage().contains("Value V1_0 of parameter iidm.export.ampl.export-version is not contained in possible values [1.0"));
 
     }
 
+    @Test
+    void writeHeadersWithVersion11() throws IOException {
+        Network network = Network.create("dummy_network", "test");
+        MemDataSource dataSource = new MemDataSource();
+
+        Properties properties = new Properties();
```

Comment:
Sonar issue:
```suggestion
        properties = new Properties();
```
There are 2 other similar Sonar issues.

Reply:
Fixed. Thank you!